.github/pull_request_template.md

```diff
@@ -1,26 +1,33 @@
-## Description
-
-<!-- A brief description of the change being made with this pull request. -->
-
 ## Motivation
 
 <!-- What inspired you to submit this pull request? -->
 
+## Changes
+
+<!-- A brief description of the change being made with this pull request. -->
+
 ## Workflow
 
-1. ⚠️⚠️ Create your PR as draft
+
+1. ⚠️ Create your PR as draft ⚠️
 2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
```

Comment:
I believe this one could be easily fix. Just prevent test to run locally if formatting fails, so people can't push unlinted code. Also, add a CI check so it cannot be merged if not formatted properly.

Reply:
>  Just prevent test to run locally if formatting fails, so people can't push unlinted code

It's not that rare that people do not run at all locally :) My feeling on that point is a pre-commit hook is the good solution

> Also, add a CI check so it cannot be merged if not formatted properly.

The job exists, but is not mandatory. -> trying to make it mandatory